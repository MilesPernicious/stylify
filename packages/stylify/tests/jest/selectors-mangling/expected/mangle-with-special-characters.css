.b{
	color: blue
}
.a::before{
	content: '\2022'
}
.a::before{
	position: absolute
}
