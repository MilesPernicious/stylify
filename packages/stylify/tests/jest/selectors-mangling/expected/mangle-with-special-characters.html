<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width">
	<meta name="generator"${$$addAttribute(Astro2.generator, "content")}>
	<title>${title}</title>
	${$$renderHead($$result)}
</head>
<body class="a"></body>
